src/coreclr/src/jit/morph.cpp

```diff
@@ -2837,10 +2846,11 @@ void Compiler::fgInitArgInfo(GenTreeCall* call)
         {
             maxRegArgs = 0;
         }
-
+#ifdef UNIX_X86_ABI
         // Add in the ret buff arg
         if (callHasRetBuffArg)
```

Comment:
The comment for `GTF_CALL_M_RETBUFFARG` says: `GT_CALL -- first parameter is the return buffer argument`;
 should it be set for `callConvIsInstanceMethodCallConv(compMethodInfoGetUnmanagedCallConv(info.compMethodInfo))` on  `defined(TARGET_WINDOWS) && !defined(TARGET_ARM)` or not? 
If yes then we need to update the comment because it is not the first argument there, if not then `callHasRetBuffArg` would not return `true` here and the check for UNIX is not necessary?